recipes/mikado/meta.yaml

```diff
@@ -0,0 +1,122 @@
+{% set name = "mikado" %}
+{% set version = "1.0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "2e668e098d465ef1bff837e152732b49bbb61f74cc5fbb938be9d72281bd710a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ version }}.{{ file_ext }}'
+  # url: https://files.pythonhosted.org/packages/f0/31/64f3630dfe4ed06b36226cf48d25370e8ee9ad875f0fb82e1dd2f129bdc2/Mikado-1.0.2.tar.gz
+  # url: file:////usr/users/ga002/venturil/workspace/mikado/dist/Mikado-1.0.2.tar.gz
+  url: https://github.com/lucventurini/mikado/archive/1.0.2.tar.gz
```

Comment:
here you can use the jinja2 variable `{{ version }}` in place of `1.0.2` for easier future maintenance